src/components/network/NewNetworkOptions.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Collapse } from 'antd';
+import DockerNetworkName from 'components/common/DockerNetworkName';
+
+interface Props {
+  setIsDockerNetworkNameValid: (value: boolean) => void;
+}
+
+const NewNetworkOptions: React.FC<Props> = ({ setIsDockerNetworkNameValid }) => {
```

Comment:
This component is so small that it feels unnecessary. You could just move this into the `NewNetwork.tsx` file.